src/util/hasSecurityAlerts.ts

```diff
@@ -6,8 +6,10 @@ type Unsubscribe = () => void
  * Returns true if the application should show the SecurityAlertsScene.
  */
 export function hasSecurityAlerts(account: EdgeAccount): boolean {
-  const { otpResetDate, pendingVouchers = [] } = account
-  return otpResetDate != null || pendingVouchers.length > 0
+  const { otpResetDate, pendingVouchers = [], username } = account
+  return (
+    username != null && (otpResetDate != null || pendingVouchers.length > 0)
+  )
```

Comment:
It should be impossible for a light account to have vouchers, so I'm not sure we need this change.

In some future bizzaro-world where we allow remote logins into light accounts, we would probably still want the "Device XYZ is trying to access your account. Deny / It was me" screen. So, I think we should put this one back.

Reply:
This change is omitting light accounts from this behavior unless I misunderstood your comment?

Reply:
I still think we need to revert this change. The `hasSecurityAlerts` method shouldn't care whether an account has a username or not.